doc/man3/X509_verify_cert.pod

```diff
@@ -2,22 +2,26 @@
 
 =head1 NAME
 
-X509_verify_cert - discover and verify X509 certificate chain
+X509_verify_cert,
+X509_verify_chain - discover and verify X509 certificate chain
 
 =head1 SYNOPSIS
 
- #include <openssl/x509.h>
+ #include <openssl/x509_vfy.h>
 
  int X509_verify_cert(X509_STORE_CTX *ctx);
+ int X509_verify_chain(X509_STORE_CTX *ctx);
 
 =head1 DESCRIPTION
 
 The X509_verify_cert() function attempts to discover and validate a
-certificate chain based on parameters in B<ctx>.
+certificate chain based on parameters in I<ctx>.
 The verification context, of type B<X509_STORE_CTX>, can be constructed
 using L<X509_STORE_CTX_new(3)> and L<X509_STORE_CTX_init(3)>.
-It usually includes a set of certificates serving as trust anchors,
-a set of non-trusted certificates that may be needed for chain construction,
+It usually includes the target certificate to be verified,
+a set of certificates serving as trust anchors,
+a list of non-trusted certificates that may be helpful for chain construction
+(where its first element, if present, serves as the default target certificate),
```

Comment:
This applies to verify_chain not verify_cert.

Reply:
Oops, this was a leftover I had forgotten to remove; done now.